system/doc/design_principles/sup_princ.md

```diff
@@ -587,21 +591,18 @@ init(_Args) ->
     {ok, {SupFlags, ChildSpecs}}.
 ```
 
-The supervisor then starts all its child processes according to the child
-specifications in the start specification. In this case there is one child
-process, `ch3`.
+Subsequently, the supervisor starts its child processes according to the child
+specifications in the start specification. In this case there is a single child
+process, called `ch3`.
 
-`supervisor:start_link` is synchronous. It does not return until all child
+`supervisor:start_link/3` is synchronous. It does not return until all child
 processes have been started.
 
 ## Adding a Child Process
 
-In addition to the static supervision tree, dynamic child processes can be added
-to an existing supervisor with the following call:
-
-```text
-supervisor:start_child(Sup, ChildSpec)
-```
+In addition to the static supervision tree, dynamic child processes
```

Comment:
Maybe somehow make it clear that "static" here means child specs as returned by `init`?